libs/platform/src/lib/table/components/table-row/table-row.component.html

```diff
@@ -156,7 +156,7 @@
 
         <ng-container *ngIf="row.state === 'readonly'; else editModeCell">
             <span
-                *ngIf="(row.value | valueByPath : column.key) === '' || (row.value | valueByPath : column.key) === null"
+                *ngIf="tableTextContainer.innerText.trim() === ''"
```

Comment:
`innerText` is `undefined` in JSDom, hence issues with unit tests, you either need to fix unit tests with mocking element innerText or do this differently

Reply:
> "There might be glitches between the detection cycles" 

Agreed but this will not be an issue for screenreader users which is what this issue is addressing